src/harpstrata/testResources/index.ts

```diff
@@ -1,18 +1,21 @@
 import type { HarpStrata } from '../types'
 import { EXAMPLE_PITCH_MATRICES } from '../Pitch/testResources'
+import { EXAMPLE_IS_ACTIVE_COMPLEXES } from '../IsActive'
 import { EXAMPLE_DEGREE_MATRICES } from '../Degree/testResources'
 import { MAJOR_DIATONIC_APPARATUS } from '../Apparatus/constants'
 
 const C_MAJOR_DIATONIC_FIRST_POZITION: HarpStrata = {
```

Comment:
This name and the other strata below need to be improved to identify the fact that we have active / inactive matrix coords now.